### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by emre-f**

### CODE REVIEW

**Review by yash-puligundla — CHANGES_REQUESTED** [review]
Hi Emre,

Looks good. Just some feedback related to the structure. 
1. Having a separate CONFIGURATION.md will make it uniform to other starter kits READMEs.
2. Consider adding a different table for each of the top-level Keys that talks about each of the attributes & their default values. I would follow the data-connect CONFIGURATION.md example, while making sure all the fields related to passport broker are added. 
3. I would remove the link to the example config file, as the contents are already displayed. Also, this way we will not have a dependency on a test file resource.

File: CONFIGURATION.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+## Configuration
+
+The Passport Broker service can be configured with custom properties, affecting behavior at runtime. There is an example config file present in this repo:
```
Comment (yash-puligundla) [inline CONFIGURATION.md:3]:
```suggestion
The Passport Broker service can be configured with custom properties, affecting behavior at runtime. Here is an example config file:
```

Comment (yash-puligundla) [inline CONFIGURATION.md]:
```suggestion
The Passport Broker service can be configured with custom properties, affecting behavior at runtime. Here is an example config file:
```

File: CONFIGURATION.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    organization:
+      name: GA4GH
+      url: https://ga4gh.org
+```
+
+This configuration file modifies some fields, but there are many more fields that can be configured.
+
+## YAML config file
+
+The YAML config file must contain a single root property of `drs`, under which all subconfigurations are located:
```
Comment (yash-puligundla) [inline CONFIGURATION.md]:
```suggestion
The YAML config file must contain a single root property of `passportBrokerConfig`, under which all subconfigurations are located:
```

File: CONFIGURATION.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+```
+brokerProps:
+    passportIssuer: http://localhost:4455/
+    brokerSecret: 1F4B4EBC23ABE9C1AFFFD5531C95F
+```
+
+| Attribute | Description | Default |
+|-----------|-------------|---------|
+| passportIssuer | URL to passport issuer organization's domain | https://ga4gh.org |
+| brokerSecret | ? | insecureSecretPleaseOverride |
```
Comment (yash-puligundla) [inline CONFIGURATION.md]:
```suggestion
| brokerSecret | The secret string used by the passport broker to mint a Passport JWT | insecureSecretPleaseOverride |
```

**Comment by yash-puligundla** [comment]
Hi @emre-f  I have added a few suggestions as in-line comments. Let me know if you have any questions

**Comment by emre-f** [comment]
@yash-puligundla are you talking about the 3 points you had above? I completed my changes regarding those points. What do you mean by "in-line comments? I couldn't see anything else.

**Comment by yash-puligundla** [comment]
Sorry @emre-f, I forgot to submit the comments. you should be able to see them now.

**Comment by emre-f** [comment]
@yash-puligundla added the changes you have suggested, they all looked correct to me.

**Review by yash-puligundla — APPROVED** [review]
Looks good, Thanks Emre. Approving & merging the PR.

**Pull request merged**